vmck.nomad

```diff
@@ -0,0 +1,63 @@
+job "vmck" {
```

Comment:
Why do we have a vmck nomad job here? It feels super overkill for a CI test. Can't you just `docker run` the vmck docker image, like we do with cluster? At the very least, it's duplicate code from the vmck repo. And it's a hell of a lot of code: the job file, the python code to talk to nomad...